nautobot/dcim/tests/test_cablepaths.py

```diff
@@ -45,6 +45,9 @@ def setUpTestData(cls):
         cls.status = cls.statuses.get(slug="connected")
         cls.status_planned = cls.statuses.get(slug="planned")
 
+        # create a Cable that is not contained in any CablePath
+        cls.dneCable = Cable(status=cls.status)
```

Comment:
is this being created intentionally as an invalid object? Normally the Cable terminations are required to be non-null and are checked for validity in the `Cable.clean()` method.

Reply:
Yes, it's the "does not exist cable". It's used in `assertContainedByPath()` to assert that `path_contains` is not returning false positives. Attempting to `.clean()` or `.save()` this Cable will fail, but it's valid to use it in filter expressions